web/src/main/webapp/hillview.css

```diff
@@ -355,8 +356,8 @@ textarea:focus, input:focus {
 }
 
 table.menu {
-    box-shadow: 4px 4px 4px rgba(0, 0, 0, 0.5);
-    border: 1px solid grey;
+    /* box-shadow: 4px 4px 4px rgba(0, 0, 0, 0.5); */
```

Comment:
Why are you changing these?

Reply:
The shadow and border make the menu appears to be shifting down when you expand the menu. 
![image](https://user-images.githubusercontent.com/7077197/88789407-57e45680-d15c-11ea-98c7-26b64ac0fa3e.png)

That image should explain how these submenus are displayed. A dummyMenu is needed when the most left column is no longer has a menu. The dummyMenu makes a submenu stay on the right column.  I displayed dummyMenu as a transparent menu. 

When a Submenu is not displayed, the display will be `none`, thus other submenus can take the space and appear to be displayed in the second column as depicted in the next image.
![image](https://user-images.githubusercontent.com/7077197/88790399-be1da900-d15d-11ea-9395-1ef008c8a0ab.png)